std/format.d

```diff
@@ -2350,8 +2482,13 @@ private void formatChar(Writer)(Writer w, in dchar c, in char quote)
         formattedWrite(w, "\\U%08X", cast(uint)c);
 }
 
-// undocumented
```

Comment:
Ilya Yaroshenko marked a few functions like that as well. And he said to me earlier this week that at least some action should be taken.